src/controllers/update/check.your.answers.controller.ts

```diff
@@ -0,0 +1,68 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import { getApplicationData } from "../../utils/application.data";
```

Comment:
I checked with @corybudd-ch and the project uses these relative paths elsewhere but it's fairly bad practice. It might be worth seeing about adding a tech debt ticket to sort the project out with absolute paths?